samples/XCompose/_.XCompose

```diff
@@ -0,0 +1,1297 @@
+# for Emacs: -*- coding: utf-8 -*-
```

Comment:
You'll need to remove this sample file and add one for filenames (in `samples/XCompose/filenames/`).